roles/test/tasks/test-importer.yml

```diff
@@ -0,0 +1,51 @@
+---
+- name: include test data
+  include_role:
+    name: sample-data
+  vars:
+    sample_role_purpose: test
+    sample_fortigate_name: fortigate_test
+    sample_config_user: fworchtest
+    sample_config_user_home: "/home/{{ sample_config_user }}"
+    import_sample_server: localhost
+
+- name: make test import
+  command: "./fworch-importer-single.pl mgm_id=1"
+  args:
+    chdir: "{{ fworch_home }}/importer"
+  become_user: "{{ fworch_user }}"
+  become: yes
+
+- name: delete test user and dir
+  user:
+    name: test
+    state: absent
+    remove: yes
+  become: yes
+
+- name: delete test fortinet gateway
+  postgresql_query:
+    db: "{{ fworch_db_name }}"
+    query: >
+      DO $do$ BEGIN 
+      DELETE FROM device WHERE dev_name='fortigate_test'; 
```

Comment:
should we use a variable for the name?

Reply:
introduced a global variable containing the name